src/tactic/explode.lean

```diff
@@ -160,6 +155,41 @@ iff_true_intro : ∀ {a : Prop}, a → (a ↔ true)
 8│1,7│ ∀I        │ a → (a ↔ true)
 9│0,8│ ∀I        │ ∀ {a : Prop}, a → (a ↔ true)
 ```
+
+In more detail:
+
+The output of `#explode` is a Fitch-style proof in a four-column diagram modeled after Metamath
+proof displays like [this](http://us.metamath.org/mpeuni/ru.html). The headers of the columns are
+"Step", "Hyp", "Ref", "Type" (or "Expression" in the case of Metamath):
+* Step: An increasing sequence of numbers to number each step in the proof, used in the Hyp field.
+* Hyp: The direct children of the current step. Most theorems are implications like `A -> B -> C`,
+  and so on the step proving `C` the Hyp field will refer to the steps that prove `A` and `B`.
+* Ref: The name of the theorem being applied. This is well-defined in Metamath, but in Lean there
+  are some special steps that may have long names because the structure of proof terms doesn't
+  exactly match this mold.
+  * If the theorem is `foo (x y : Z) : A x -> B y -> C x y` the Ref field will contain `foo`, `x`
+    and `y` will be suppressed (because term construction is not interesting), while the Hyp field
```

Comment:
This is a badly formed sentence and I'm not sure what was intended. What's contained, what's being suppressed?

Reply:
The step that applies `foo` will say `foo` in the Ref field, and there will be 2 hypotheses in the Hyp field (instead of 4 as you might expect given that foo has 4 arguments). Rewording welcome

Reply:
Oh, I get it, I read "foo, x and y" as a list without an Oxford comma.

Reply:
```suggestion
  * If the theorem is `foo (x y : Z) : A x -> B y -> C x y`, the Ref field will contain `foo`;
   `x` and `y` will be suppressed, because term construction is not interesting; 
   and the Hyp field
```

Reply:
I ended up splitting this sentence into multiple bullet points. Hopefully that makes it more clear.